### PR TITLE
chore: use node parseArgs instead of minimist

### PR DESCRIPTION
**Pull request by kevcodez**
Node 18 comes with a built-in args parser - no need for another dependency for these simple use-cases

### CODE REVIEW

File: apps/docs/generator/index.ts [hunk 1]
```diff
@@ -3,28 +3,42 @@ import CliGenerator from './cli'
 import ConfigGenerator from './config'
 import SdkGenerator from './sdk'
 import LegacyGenerator from './legacy'
-import * as minimist from 'minimist'
+import { parseArgs } from 'node:util'
+const assert = require('node:assert')
```
Comment (gregnr) [inline apps/docs/generator/index.ts:7]: Any reason this needed to be `require` over `import`?

File: spec/parser/tsdoc.ts [hunk 2]
```diff
@@ -1,28 +1,25 @@
 import { writeToDisk } from './helpers'
 import stringify from 'json-stringify-safe'
+import { parseArgs } from 'node:util'
+import fs from 'fs'
+const assert = require('node:assert')
```
Comment (gregnr) [inline spec/parser/tsdoc.ts:5]: Here as well

**Review by gregnr — APPROVED** [review]
Supa nice 👍

File: apps/docs/generator/index.ts (hunk 1 above)
Comment (kevcodez) [reply apps/docs/generator/index.ts]: There were some weird type errors + execution issues, but I now found a working way with es6 imports

**Pull request merged**